watertap/costing/watertap_costing_package.py

```diff
@@ -1058,3 +1100,35 @@ def cost_by_flow_volume(blk, flow_cost, flow_to_cost):
     blk.capital_cost_constraint = pyo.Constraint(
         expr=blk.capital_cost == blk.flow_cost * flow_to_cost
     )
+
+
+def cost_uv_aop_stack(blk, flow_cost, electricity_cost, factor_uv_replacement):

```

Comment:
Not sure `uv_aop_stack` is the nomenclature that best represents the UV reactor configuration. ED is costed as such because it seems to be completely modular, so the stack of cells makes sense. Here, something like `uv_aop_lamp_bundle` would be a physical equivalent, but the model is not being costed as such and therefore I'm not sure of the reason this is presented as its own function, similar to the ED costing. Maybe you and others can comment for clarification.

Reply:
I've changed to `cost_uv_aop_bundle` as suggested, the UV system include uv reactor, lamps and other equipment. I previously thought "stack" has a similar meaning with "bundle", but I agree "bundle" is more appropriate in this case.